cmd/dsl/main.go

```diff
@@ -338,13 +393,13 @@ func (s Simulator) execute() {
 			arg := strings.Split(instr.getSecond(), "@")
 			dst, seq := arg[0], arg[1]
 			srcPuppet := s.getSrcPuppet()
-			dstPuppet := s.puppetMap[dst]
+			dstPuppet := s.getPuppet(dst)
 			err := DoHast(srcPuppet, dstPuppet, seq)
 			s.evaluateRun(err)
 		default:
 			// unknown command, abort test run
-			instr.TestAbort(errors.New("Unknown simulator command"))
-			s.exit()
+			s.Abort(errors.New("Unknown simulator command"))
+			continue
```

Comment:
this `continue` here is a bit weird (seems to imply Abort() isn't fatal)


```suggestion
			return
```

Reply:
ya i had continue here cause i wanted the loop to freeze at the channel select stmt, but i think i might have been mistaken. will fix in the latest PR!